Tasks/UseRubyV1/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "use-ruby",
+  "version": "0.0.0",
+  "description": "Use Ruby Ecosystem",
+  "main": "useruby.js",
+  "scripts": {
+    "test": "mocha Tests/L0.js",
+    "build": "tsc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "keywords": [
+    "Azure",
+    "Pipelines",
+    "Tasks",
+    "Ruby"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "azure-pipelines-task-lib": "^2.8.0",
+    "azure-pipelines-tool-lib": "^0.12.0"
+  },
+  "devDependencies": {
```

Comment:
You shouldn't have devDependencies here, they should exist as dependencies in a separate package.json inside the tests folder (that's why the build is currently failing).

Also, not sure you need all of these.

Reply:
👍 Made those updates